.github/workflows/ci.yml

```diff
@@ -74,6 +103,15 @@ jobs:
         env:
           NX_AGENT_NAME: ${{ matrix.agent }}
 
+      - name: Revoke and delete GPG key
+        run: |
```

Comment:
We would want this to always run, even in error cases, right? So we need to add the if always syntax to this step